public/app/features/dashboard/components/GenAI/GenAIDashDescriptionButton.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 
+import { llms } from '@grafana/experimental';
+
 import { DashboardModel } from '../../state';
 
 import { GenAIButton } from './GenAIButton';
-import { Message, Role } from './utils';
+import { Role } from './utils';
```

Comment:
Do we have the Role also exported from `llm`?

Reply:
We do have `Role` exported from `llm` but unfortunately it is just a string literal union type which as far as I am aware we are unable to convert in place to an enum (I believe you can convert an enum to a string literal union type but not the other way around)
```
type openai_d.Role = "function" | "system" | "user" | "assistant"
```